README.md

```diff
@@ -1069,6 +1070,66 @@ The initialization function is run from the context of the HTML element.
 It should be noted that children nodes are removed from the DOM when widget instantiation occurs, and added as children
 to the widget instance.
 
+#### Meta
+
+Widget meta is used to access information that you would normally only have access to on a DOM element. For example, the dimensions of an HTML node.  You can access and respond to meta data during a widget's render operation.
```

Comment:
I know that this has a lot of features specific to DOM, but could/would we ever recommend using this for non-DOM meta?